specification/scvmm/ScVmm.Management/examples/2024-06-01/GuestAgents_ListByVirtualMachineInstance_MaximumSet_Gen.json

```diff
@@ -0,0 +1,43 @@
+{
+  "title": "GuestAgents_ListByVirtualMachineInstance_MaximumSet",
+  "operationId": "GuestAgents_ListByVirtualMachineInstance",
+  "parameters": {
+    "api-version": "2024-06-01",
+    "resourceUri": "gtgclehcbsyave"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "properties": {
+              "uuid": "hbsgztyakewtgbuxbesezncnzu",
+              "credentials": {
+                "username": "jqxuwirrcpfv"
+              },
+              "httpProxyConfig": {
+                "httpsProxy": "uoyzyticmohohomlkwct"
+              },
+              "provisioningAction": "install",
+              "status": "jpoukrzfenzrmjdahimkl",
+              "customResourceName": "mhqymxkapuvsugd",
+              "provisioningState": "Succeeded"
```

Comment:
nit: you can add "privateLinkScopeResourceId" here as well.

Reply:
Done